server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -1798,10 +1798,15 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
      * @param signalMessage - Ticketed join signal message
      */
     private addSequencedSignalClient(clientJoinMessage: IClientJoin, signalMessage: ISignalMessageOutput) {
+        const op = signalMessage.message.operation;
```

Comment:
@GaryWilber, please take a look if this is correct (including services like FRS that do not fully support sequenced signals).

Overall feedback: I believe both fields should move from ISignalMessage to ISignalClient. Here are couple reasons:
- Most important - client does not have that information when processing IConnected.initialClients. Without that data, it's impossible for clients to reason about any order here (even with this info it's really hard due to non-reliable signal delivery, but that makes it almost impossible to do anything).
- It would make some sense to keep it on ISignalMessage if other signals could receive this info. For example, if client could send some special signal that would roundtrip through Deli, such that it can always learn about "tip" (atomically increment and read clientConnectionNumber). But I'd guess we are not planning to have something like that, so having it on ISignalMessage  does not make much sense.